packages/eslint-plugin/src/rules/no-invalid-void-type.ts

```diff
@@ -29,6 +32,10 @@ export default util.createRule<[Options], MessageIds>({
       invalidVoidNotReturnOrGeneric:
         'void is only valid as a return type or generic type variable',
       invalidVoidNotReturn: 'void is only valid as a return type',
+      invalidVoidNotReturnOrThisParam:
+        'void is only valid as return type or type of this parameter',
```

Comment:
should have code quotes:
```suggestion
        'void is only valid as return type or type of `this` parameter',
```